### PR TITLE
[TASK] Store trustline settings in blob (RT-2862)

### PR DESCRIPTION
**Pull request by darkdarkdragon**
Store advanced trustline settings in the user blob,
not browser's local storage

### CODE REVIEW

File: src/js/services/id.js [hunk 1]
```diff
@@ -145,6 +145,12 @@ module.factory('rpId', ['$rootScope', '$location', '$route', '$routeParams', '$t
     },true);
 
     $scope.$on('$blobUpdate', function(){
```
Comment (rht) [inline src/js/services/id.js]: This is unnecessary / redundant with trust.js#L33.

File: src/js/services/id.js [hunk 2]
```diff
@@ -145,6 +145,12 @@ module.factory('rpId', ['$rootScope', '$location', '$route', '$routeParams', '$t
...
+      var d = $scope.userBlob.data;
+      Options.advanced_feature_switch = !!(d.clients &&
+          d.clients.rippletradecom &&
+          d.clients.rippletradecom.trust &&
+          d.clients.rippletradecom.trust.advancedMode);
+      $scope.advanced_feature_switch = Options.advanced_feature_switch;
```
Comment (rht) [inline src/js/services/id.js]: This is unnecessary / redundant with trust.js#L33

File: src/js/tabs/trust.js [hunk 3]
```diff
@@ -27,7 +27,11 @@ TrustTab.prototype.angular = function (module)
                                   function ($scope, books, $timeout, $routeParams, id,
                                             $filter, $network, $rpTracker, keychain)
   {
-    $scope.advanced_feature_switch = Options.advanced_feature_switch;
+    
+    updateAdvancedModeSwitch();
```
Comment (rht) [inline src/js/tabs/trust.js]: Wrap this with ```$scope.$on('!$scope.loadingAccount && !$scope.account.Balance && connected', function () { ...}``` to wait until the blob is available.

See https://github.com/rht/ripple-client/commit/33b27331b08cc965f5c12e4eaf45485732a77298

**Comment by annatonger** [comment]
I took this off hold. @yongsoo @MatthewPhinney @mrajvanshy @vhpoet

**Comment by vhpoet** [comment]
LGTM

**Comment by mrajvanshy** [comment]
LGTM

**Pull request merged**